README.md

```diff
@@ -175,6 +177,78 @@ that in the example above: Rails has used the `MovieSerializer` to render the
 `movie` JSON, so all of the attributes we listed in that serializer are rendered
 in the `Director`'s `index` and `show` routes.
 
+## Serializing a Many-to-Many Associations
+
+Our Movie example uses a one-to-many association (directors have many movies and
+movies belong to a director), but you can use the same setup with a many-to-many
```

Comment:
Suggestion: "you can also use Active Model Serializers with a many-to-many association" instead of "you can use the same setup with a many-to-many association"